elasticsearch-jmeter/azuredeploy.json

```diff
@@ -0,0 +1,426 @@
+            "allowedValues": [
+                1,
+                2,
+                3,
+                4,
+                5,
+                6,
+                7,
+                8,
+                9
+            ],
+            "metadata": {
+                "description": "Number of subordinate JMeter nodes to provision"
+            }
+        },
+        "subNodeSize": {
+            "type": "string",
+            "defaultValue": "Standard_D2_v2",
+            "allowedValues": [
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D4_v2",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7"
+            ],
+            "metadata": {
+                "description": "Size of the subordinate JMeter nodes"
+            }
+        },
+        "bossNodeSize": {
+            "type": "string",
+            "defaultValue": "Standard_D2_v2",
+            "allowedValues": [
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D4_v2",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4"
+            ],
+            "metadata": {
+                "description": "Size of the boss JMeter node"
+            }
+        },
+        "jarball": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch-jmeter/resources/jarball.zip",
+            "metadata": {
+                "description": "The location of the test library and jar dependencies. This is extracted to every node under /opt/jmeter/apache-jmeter-2.13/lib/junit"
+            }  
+        },
+        "testpack": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch-jmeter/resources/testpack.zip",
+            "metadata": {
+                "description": "The location of the test jmx and run properties. This is extracted to the JMeter master node only, in /opt/jmeter"
+            }  
+        },
+        "esClusterName": {
+            "type": "string",
+            "defaultValue": "elasticsearch",
+            "metadata": {
+                "description": "The name of the Elasticsearch cluster to target"
+            }
+        },
+        "templateBase": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master",
+            "metadata": {
+                "description": "Change this value to your repo name if deploying from a fork"
+            }
+        }
+    },
+    "variables": {
+        "templateBaseUrl": "[concat(parameters('templateBase'), '/elasticsearch-jmeter/')]",
+        "locationMap": {
+            "ResourceGroup": "[resourceGroup().location]",
```

Comment:
neat trick! but I don't know if you need to do so much work when new locations are introduced instead of just defaulting to resourceGroup location?